security_routes/admin_routes.py

```diff
@@ -95,6 +95,17 @@ async def create_group(
     return return_group
 
 
+@router.get("/groups", response_model=list[Group])
+async def get_groups(
```

Comment:
looks pretty simple but i think we should a test in the admin flow of test admin routes, it also serves as documentation

Reply:
OK yep. I've now added a test. I'm a bit scared to run it before I've merged your updates! So I think I'll merge this, and if it fails the test I'll fix it.